types/clmtrackr/index.d.ts

```diff
@@ -0,0 +1,55 @@
+// Type definitions for clmtrackr 1.1
+// Project: https://github.com/auduno/clmtrackr
+// Definitions by: hellochar <https://github.com/hellochar>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface TrackerParams {
+    /** whether to use constant velocity model when fitting (default is true) */
+    constantVelocity?: boolean;
+    /** the size of the searchwindow around each point (default is 11) */
+    searchWindow?: number;
+    /** whether to use webGL if it is available (default is true) */
+    useWebGL?: boolean;
+    /** threshold for when to assume we've lost tracking (default is 0.50) */
+    scoreThreshold?: number;
+    /** whether to stop tracking when the fitting has converged (default is false) */
+    stopOnConvergence?: boolean;
+    /** object with parameters for facedetection : */
+    faceDetection?: {
+        /** whether to use web workers for face detection (default is true) */
+        useWebWorkers?: boolean;
+    };
+}
+
+type IPosition = [number, number];
+
+type Model = any;
+
+declare namespace _default {
```

Comment:
When I require the module I just see `tracker` and `version` properties directly, with no `default` property. This is on node.js using:
```ts
//hax to make it not crash on load
global.window = global;
window.Event = class Event {}
console.log(require("clmtrackr"));
```
Could you check if your module loader is injecting a default export where there is none?